examples/lock-app/lock-common/lock-app-wifi.zap

```diff
@@ -0,0 +1,6343 @@
+{
```

Comment:
Should this be in lock-common or in lock-app/silabs/..

The matching .matter file is missing too